doc/nrf/known_issues.rst

```diff
@@ -3042,7 +3045,37 @@ NCSDK-9992: Multiple extra CMake options applied as single option
 
   **Workaround:** Create a CMake preload script containing ``FOO`` and ``BAR`` settings, and then specify ``-C <pre-load-script>.cmake`` in :guilabel:`Extra CMake Build Options`.
 
-----
+.. _known_issues_vulnerabilities:
 
-In addition to these known issues, check the current issues in the `official Zephyr repository`_, since these might apply to the |NCS| fork of the Zephyr repository as well.
-To get help and report issues that are not related to Zephyr but to the |NCS|, go to Nordic's `DevZone`_.
+Security vulnerabilities
```

Comment:
Does this belong to the Security page more? May be divide the existing page into subpages and add this as a sub page there?